app/services/witter.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function weet() {
+  try {
+    const response = await axios.get('https://quote-garden.herokuapp.com/api/v3/quotes/random');
+    return response.data.data;
+  } catch (error) {
+    return { message: 'Error en el servidor', error };
```

Comment:
Add a log here with the possible error thrown by the API, for that you could use this logger `app/logger/index.js` that we have within the project.

Reply:
To return the error to the client add a custom error, look at the ones we have here `app/errors.js`, also do not forget to add the appropriate `statusCode` here `app/middlewares/errors.js`